mmstudio/src/main/java/org/micromanager/acquisition/SequenceSettings.java

```diff
@@ -211,10 +221,11 @@ public SequenceSettings build() {
    public ArrayList<Double> customIntervalsMs = null;
    /**
     * an array of ChannelSpec settings (one for each channel)
+    * no member of the array should ever by null
     * @deprecated use Builder and channels() instead
     */
    @Deprecated
-   public ArrayList<ChannelSpec> channels = new ArrayList<>();
+   private ArrayList<ChannelSpec> channels = new ArrayList<>();
```

Comment:
Was this perhaps left `private` by accident, given your description (and the remaining `@Deprecate`)?